portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -16,29 +16,51 @@
 
 import com.google.sps.data.Comment;
 import com.google.gson.Gson;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
 import java.io.IOException;
 import java.util.*;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data") //"data" kept loading a blank screen no matter what I did so this was my workaround
+@WebServlet("/data") 
 public class DataServlet extends HttpServlet {
+  // declare global constants so as to not incur the risks of hard-coding strings
```

Comment:
This comment is unnecessary. Also make these constants, private final and all upper case